app/scripts/pages/logs.js

```diff
@@ -11,7 +10,7 @@ var Logs = React.createClass({
     host: React.PropTypes.string
   },
   getInitialState: function () {
-    var t = this
+    // var t = this
     var req = null
     // TODO: fix this
```

Comment:
There is just so much to do when I even look at a single component in here that I could create 3 issues from that alone..so I haven't started with that